tests/data_context/test_data_context_v013.py

```diff
@@ -245,49 +245,6 @@ def test_config_variables(empty_data_context):
     assert set(context.config_variables.keys()) == {"instance_id"}
 
 
-@pytest.mark.filterwarnings(
-    "ignore:get_batch is deprecated*:DeprecationWarning:great_expectations.data_context.data_context"
-)
-def test_get_batch_of_pipeline_batch_data(empty_data_context, test_df):
```

Comment:
@donaldheppner I do not intuitively see why this test is being deleted (I thought that we are deprecating V2, not yet V3). Could you please help with a justification? Thank you.

Reply:
These tests check the RuntimeBatchRequest constructor for the same logic that is tested other places and in test_batch.py -- the actual assertions here have nothing to do with Spark.